frontend/src/views/shop/AddShop.vue

```diff
@@ -130,11 +135,39 @@
                 </button>
             </div>
         </vee-form>
+
+        <!-- Plugin Connection Modal -->
+        <modal :show="showPluginModal" close-x-mark @close="closePluginModal">
+            <template #title>
+                Connect using Shopmon Plugin
+            </template>
+            <template #content>
+                <p>Paste the base64 string from your Shopmon Plugin:</p>
+                <textarea
+                    v-model="pluginBase64"
+                    class="field"
+                    rows="4"
+                    placeholder="eyJ1cmwiOiJodHRwczpcL1wvZGVtby5mb3MuZ2ciLCJjbGllbnRJZCI6..."
+                ></textarea>
+                <div v-if="pluginError" class="field-error-message">
+                    {{ pluginError }}
+                </div>
+            </template>
+            <template #footer>
+                <button type="button" class="btn btn-primary" @click="processPluginData">
+                    Import Data
+                </button>
+                <button type="button" class="btn btn-cancel" @click="closePluginModal">
+                    Cancel
+                </button>
+            </template>
+        </modal>
     </main-container>
 </template>
 
 <script setup lang="ts">
 import { useAlert } from '@/composables/useAlert';
+import Modal from '@/components/layout/Modal.vue';
```

Comment:
The AI already does this but templates are auto loaded

Reply:
fixed in https://github.com/FriendsOfShopware/shopmon/pull/509/commits/4f527d2aa0192ef24d9d819f73c58fd4cb9d4425